plugins/identities/http.go

```diff
@@ -30,6 +33,10 @@ type HTTPAuthConfig struct {
 
 	// bearer
 	Token string `mapstructure:"token,omitempty" json:"token,omitempty" yaml:"token,omitempty" validate:"required_if=Type bearer"`
+
+	// oidc
+	Audience                        string `mapstructure:"audience,omitempty" json:"audience,omitempty" yaml:"audience,omitempty" validate:"required_if=Type oidc"`
+	GoogleServiceAccountCredentials string `mapstructure:"google_service_account_credentials,omitempty" json:"google_service_account_credentials,omitempty" yaml:"google_service_account_credentials,omitempty" validate:"required_if=Type oidc"`
```

Comment:
If we say OIDC as [Open ID Connect](https://openid.net/developers/how-connect-works/) I think we should keep it generic to avoid misinterpreted definition. If we require a google service account to do this flow then it becomes google-specific flow.
I think it would be better to either replace and use the generic flow one or rename this with some google specific naming.

Reply:
updated to `google_idtoken`